React/Exercise/src/pages/Ex6.tsx

```diff
@@ -0,0 +1,26 @@
+import Animal from '../components/Animal'
+import Header from '../components/Header'
+
+const animals: string[]= ['dog', 'cat', 'chicken', 'cow', 'sheep', 'horse']
+const MapListComponent = () => {
+   
+  return (
+    <>
+        <Header />
+        <main className='container mx-auto px-8 pt-20'>
+            <section className='mt-10'>
+                <ul className='list-disc mx-8 marker:text-yellow-500 text-xl'>
+                    {
+                        animals.map((animal, index) =>(
+                            <Animal key={index} animal={animal} />
+                        ))
+                    }
+
+                </ul>
```

Comment:
Hey @riyoneri, I'm not sure I understand this comment because I have a component named Animal where by a passed the element of the map. Or do I need to do it differently???